packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1747,14 +1748,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             this.consecutiveReconnects++;
 
             if (!this.shouldContinueReconnecting()) {
-                this.closeFn(new GenericError(
-                    // pre-0.58 error message: MaxReconnectsWithNoProgress
```

Comment:
Please leave this comment, it will help someone searching the code for this error message when looking at telemetry from older clients.